ledger-core/virtual/object/object_migration_test.go

```diff
@@ -66,12 +67,17 @@ func TestSMObject_MigrationCreateStateReport_IfStateMissing(t *testing.T) {
 		LogMock.Return(smachine.Logger{}).
 		ShareMock.Return(smachine.NewUnboundSharedData(&report)).
 		PublishMock.Set(func(key interface{}, data interface{}) (b1 bool) {
```

Comment:
That publishMockFn became bigger and more complicated.
Can you extract it before migrationCtx is created and, also, cheeck that ReportKey AND SummarySyncKey both called. Also default - we should check other cases.

Reply:
Fixed